app/helpers/navigation_items.rb

```diff
@@ -74,7 +74,15 @@ def for_provider_account_nav(current_provider_user, current_controller, performi
       items = []
 
       unless performing_setup
-        items << NavigationItem.new(t('page_titles.provider.account'), provider_interface_account_path, active?(current_controller, %w[account profile provider_users organisations provider_relationship_permissions]))
+        if FeatureFlag.active?(:accredited_provider_setting_permissions)
+          if current_provider_user.authorisation.can_manage_users_or_organisations_for_at_least_one_setup_provider?
+            items << NavigationItem.new(t('page_titles.provider.organisation_settings'), provider_interface_organisation_settings_path, active?(current_controller, %w[organisations provider_users]))
```

Comment:
We should add the organisation_settings controller to the active check here

Reply:
Good catch